odd_collector/adapters/postgresql/mappers/connectors.py

```diff
@@ -0,0 +1,55 @@
+import contextlib
```

Comment:
Move connector away to higher directory, because  it isn't a part of mappers package